src/main/java/world/anhgelus/molehunt/Molehunt.java

```diff
@@ -139,11 +139,27 @@ public void onInitialize() {
                     return Command.SINGLE_SUCCESS;
                 })
         ));
-        command.then(literal("moles").requires(source ->
-                (game != null && game.isAMole(source.getPlayer())) ||
-                        (source.getPlayer() != null && source.getPlayer().isSpectator())
-        ).executes(context -> {
-            context.getSource().sendFeedback(() -> Text.translatable("commands.molehunt.moles.list").append(" " + game.getMolesAsString()),false);
+        command.then(literal("role").requires(ServerCommandSource::isExecutedByPlayer).executes(context -> {
+            if (game == null || !game.hasStarted()) {
+                throw (new SimpleCommandExceptionType(Text.translatable("commands.molehunt.stop.failed"))).create();
```

Comment:
We should rename this one into `commands.molehunt.command_failed` or something like that

Reply:
yeah you're right